MyPerfectOnboardingApplication/Sources/MyPerfectOnboarding.Database/DatabaseBootstraper.cs

```diff
@@ -1,14 +1,23 @@
-﻿using MyPerfectOnboarding.Contracts;
+﻿using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Serializers;
 using MyPerfectOnboarding.Contracts.Database;
+using MyPerfectOnboarding.Contracts.Dependency;
+using MyPerfectOnboarding.Contracts.Models;
+using MyPerfectOnboarding.Database.Models;
 using MyPerfectOnboarding.Database.Repository;
-using Unity;
-using Unity.Lifetime;
 
 namespace MyPerfectOnboarding.Database
 {
     public class DatabaseBootstraper : IBootstraper
     {
-        public IUnityContainer RegisterTypesTo(IUnityContainer container)
-            => container.RegisterType<IListRepository, ListRepository>(new HierarchicalLifetimeManager());
+        static DatabaseBootstraper()
+        {
+            //It's needed because of the tests. They failed cause ctor was called before every test, but the serializer were already registered.
```

Comment:
First things first:
```suggestion
            // It's needed because of the tests. They failed cause ctor was called before every test, but the serializer were already registered.
```
🤣

Reply:
It is not "needed **because of** tests". It is needed because `BsonSerilizer` can only be set once in application's lifetime (and nothing prevents this class from being re-instantiated). That's the distinction between changing the code for the sake of tests and for the sake of clarity I have been speaking about in person. 

Your role as a developer is to help users of your code not to misuse it by accident and this is (kinda) exactly what your test did. I like that you described a scenario where there might be a problem (~ second sentence), however, the first sentence needs to contain the root-cause 😃 

On sentence structure, don't use `ctor` and the `serializer was` (rather than `serializer were`). But I guess you will restructure the comment anyway 😄